pkgs/top-level/aliases.nix

```diff
@@ -2,9 +2,21 @@ self:
 
 with self;
 
+let
+  # Removing recurseForDerivation prevents derivations of aliased attribute
+  # set to appear while listing all the packages available.
+  removeRecurseForDerivations = _n: alias: with lib;
+    if alias.recurseForDerivations or false then
+      removeAttrs alias ["recurseForDerivations"]
+    else alias;
```

Comment:
Shouldn't `removeAttrs` return the same set if nothing has changed ? In any case this is not new code so it can stay the same.